src/main/webapp/WEB-INF/tags/navBarBrand.tag

```diff
@@ -0,0 +1,5 @@
+<%@ tag description="TEAMMATES logo brand"%>
```

Comment:
`navBarBrand` is a name @whipermr5 suggested in a previous PR. I will rename it then.